src/main/java/datasystem/DataSystemServer.java

```diff
@@ -0,0 +1,45 @@
+package datasystem;
```

Comment:
The data system server properly hosts a data system implementation to be accessed through grpc calls.